src/v0/destinations/braze/transform.js

```diff
@@ -236,6 +246,10 @@ function processTrackWithUserAttributes(message, destination, mappingJson, proce
       );
       if (dedupedAttributePayload) {
         requestJson.attributes = [dedupedAttributePayload];
+      } else if (reqMetadata?.features && reqMetadata.features[FEATURE_FILTER_CODE]) {
```

Comment:
Take this out before conditional using destructuring

Reply:
addressed